win32/src/job_object_helper.c

```diff
@@ -11,7 +11,130 @@
 
 IMPLEMENT_MOCKABLE_FUNCTION(, int, job_object_helper_limit_resources, uint32_t, percent_physical_memory, uint32_t, percent_cpu)
 {
-    (void) percent_physical_memory;
-    (void) percent_cpu;
-    return MU_FAILURE;
+    int result;
+    LogInfo("job_object_helper_limit_resources called with parameters:"
```

Comment:
>LogInfo [](http://example.com/codeflow?start=4&length=7)

georeplication-ism detected.

Only georeplication logs extensively parameters at every function entry/exit. c-pal and all the way up to EBS they do not.

Remove plz.